contrib/qat/compression/qatzstd/compressor/source/qatzstd_compressor_impl.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "envoy/compression/compressor/compressor.h"
+#include "envoy/server/factory_context.h"
+
+#include "source/common/common/logger.h"
+#include "source/common/compression/zstd/common/base.h"
+#include "source/common/compression/zstd/common/dictionary_manager.h"
+#include "source/common/compression/zstd/compressor/zstd_compressor_impl_base.h"
+
+#include "qatseqprod.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Compression {
+namespace Qatzstd {
+namespace Compressor {
+
+using ZstdCDictManager =
+    Envoy::Compression::Zstd::Common::DictionaryManager<ZSTD_CDict, ZSTD_freeCDict,
+                                                        ZSTD_getDictID_fromCDict>;
+using ZstdCDictManagerPtr = std::unique_ptr<ZstdCDictManager>;
```

Comment:
can we reuse those https://github.com/envoyproxy/envoy/pull/32166/files#diff-52dd0ee0143d38ea0deb06569c98df06fa1ebe7dbc50ee97b566e252b3be7215R13-R15 ?